### PR TITLE
fix(server): fix validation issues

### PR DESCRIPTION
**Pull request by piyushagwl007**
## PR Checklist
Please check that your PR fulfills the following requirements:

- [x] The commit message follows [our guidelines](https://github.com/pesto-students/batch-9-Lumen/wiki/Commit-message-convention)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] GIFs/Videos/Images added (only for frontend changes)

## PR Type
What kind of change does this PR introduce? 

- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Infrastructure changes
- [ ] Other... Please describe:
Issue Number: 
#81 

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

File: server/src/utils/validations/addErrorIfNotValid.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
```
Comment (nimish-gupta) [inline server/src/utils/validations/addErrorIfNotValid.js]: new line?

**Review by nimish-gupta — APPROVED** [review]

File: server/src/utils/validations/addErrorIfNotValid.js (hunk 1 above)
Comment (piyushagwl007) [reply server/src/utils/validations/addErrorIfNotValid.js]: removed :smile:

**Review by nihalanisumit — APPROVED** [review]
LGTM

**Review by karansapolia — APPROVED** [review]
LGTM

**Pull request merged**